helm-charts/seldon-core-operator/templates/deployment_seldon-controller-manager.yaml

```diff
@@ -129,6 +129,8 @@ spec:
           value: '{{ .Values.executor.requestLogger.defaultEndpoint }}'
         - name: DEFAULT_USER_ID
           value: '{{ .Values.defaultUserID }}'
+        - name: KEDA_ENABLED
+          value: '{{ .Values.keda.enabled }}'          
         image: '{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}'
```

Comment:
Did you change this template by hand?
It needs to be auto-generated by running `cd operator/helm && make create`

Reply:
Thank you! fixed now.